brightlayer-ui/doc-it
# Feature/2971 update announcement api

Fixes 2971 .


#### Changes proposed in this Pull Request:
- Update API URL
- Add dompurify package
- 


#### Screenshots:


#### To Test:
Update following code :

Line no: src/app/components/announcements/announcementAppbar.tsx:42
const checkDateRange = (data: AnnouncementData): boolean => {
        /* eslint-disable */
        const currentDate = new Date().toJSON().slice(0, 10);
        const from = new Date(data?.startDate);
        const to = new Date(data?.endDate);
        const check = new Date(currentDate);
        /* eslint-disable */
        return true;
 };
